src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,21 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Random random = new Random();
+
+    public Random getRandom() {
+        return random;
+    }
+
+    public void setRandom(Random random) {
+        this.random = random;
+    }
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Color.values().length);
+        Color color = Color.values()[index];
+        return color.toString();
```

Comment:
checklist issue, use .name()

Reply:
what do you mean by checklist? didn't understand